<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1608</width>
    <height>1104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Explorer</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,5" columnstretch="1,5">
    <item row="1" column="0">
     <widget class="QTreeView" name="dirtree_view">
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QStackedWidget" name="main_panel_stacked">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="icon_mode_page">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListView" name="main_list_view">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="movement">
           <enum>QListView::Movement::Static</enum>
          </property>
          <property name="resizeMode">
           <enum>QListView::ResizeMode::Adjust</enum>
          </property>
          <property name="viewMode">
           <enum>QListView::ViewMode::IconMode</enum>
          </property>
          <property name="uniformItemSizes">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="list_mode_page">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="main_table_view">
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>30</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="top_panel" rowstretch="1,0,0" columnstretch="1,3,1,1">
      <item row="0" column="3">
       <widget class="QComboBox" name="icon_size_combo_box">
        <item>
         <property name="text">
          <string>Tiny</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Small</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Large</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Extra large</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="view_type_combo_box">
        <item>
         <property name="text">
          <string>Minimal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Extended</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="check_box_size">
          <property name="text">
           <string>Size</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_box_type">
          <property name="text">
           <string>Type</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_box_date">
          <property name="text">
           <string>Last modified</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_box_perms">
          <property name="text">
           <string>Permissions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_box_created">
          <property name="text">
           <string>Created</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_box_owner">
          <property name="text">
           <string>Owner</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1" alignment="Qt::AlignmentFlag::AlignBottom">
       <widget class="QLineEdit" name="path_line">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="3" alignment="Qt::AlignmentFlag::AlignTop">
       <widget class="QSpinBox" name="icon_size_spinbox"/>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="go_arrow_layout" stretch="0,0,1,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignBottom">
         <widget class="QPushButton" name="go_home_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="go-home"/>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignBottom">
         <widget class="QPushButton" name="go_back_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="go-previous"/>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignBottom">
         <widget class="QPushButton" name="go_next_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="go-next"/>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="save_settings_button">
          <property name="text">
           <string>Save current settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_def_dir">
          <property name="text">
           <string>Make current directory default</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="change_def_setts">
        <property name="text">
         <string>Change default settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
